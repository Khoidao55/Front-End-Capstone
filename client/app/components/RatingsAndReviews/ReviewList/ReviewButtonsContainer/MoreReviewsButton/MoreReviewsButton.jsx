import React from 'react';

const MoreReviewsButton = ({ visibleReviews, setVisibleReviews}) => (
  <button onClick={() => setVisibleReviews(visibleReviews + 2)} type="button" className="moreReviewsButton">more reviews</button>
);

export default MoreReviewsButton;
