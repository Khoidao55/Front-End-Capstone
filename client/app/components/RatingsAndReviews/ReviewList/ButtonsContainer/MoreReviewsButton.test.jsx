import React from 'react';
import { shallow } from 'enzyme';
import MoreReviewsButton from './MoreReviewsButton.jsx';

describe('"more reviews" button: ', () => {
  test('it renders a button', () => {
    const wrapper = shallow(<MoreReviewsButton />);

    expect(wrapper).toMatchSnapshot();
  });

  // TODO - add functionality test to button
});
