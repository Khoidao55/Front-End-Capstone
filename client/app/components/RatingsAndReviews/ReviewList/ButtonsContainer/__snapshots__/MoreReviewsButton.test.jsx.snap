// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`"more reviews" button:  it renders a button 1`] = `ShallowWrapper {}`;
